packages/api/src/cli.ts

```diff
@@ -1,14 +1,23 @@
 #!/usr/bin/env node
 
+import makeStreamInfoSvc from "./app/stream-info/stream-info-app";
 import "./dotenv";
 import makeApp from "./index";
-import makeStreamInfoSvc from "./app/stream-info/stream-info-app";
+import activeCleanup from "./jobs/active-cleanup";
+import createDbTables from "./jobs/create-db-tables";
+import updateUsage from "./jobs/update-usage";
 import parseCli from "./parse-cli";
 
 if (require.main === module) {
   const args = parseCli();
   if (args.streamInfoService) {
     makeStreamInfoSvc(args);
+  } else if (args.job === "active-cleanup") {
+    activeCleanup(args);
+  } else if (args.job === "create-db-tables") {
+    createDbTables(args);
```

Comment:
I think I don't understand the idea of `createDbTables()`. Why do we want to run a separate `initContainer` to create DB tables?

Reply:
By running an `initContainer` we can have a more controlled environment for creating the tables and indexes, which has been taking a long time these days. I think the main advantages are:
- Simpler healthchecks for the main container. After https://github.com/livepeer/livepeer-infra/pull/2097 we changed the `startupProbe` config to allow the API to take up to 2 minutes to start with a granularity (check period) of 10 seconds. With the `initContainer` we will be done with the startup as soon as it's done.
- Quicker turnaround in case of container crashes. I think this is the biggest benefit. Today, if an API container crashes for any reason, it will start again and try to do an expensive operation which is creating tables and indexes on the DB all over again. This often causes crash loops when there is some network issue going on, since the chance of getting more errors on that startup is increased. With an `initContainer` (and `create-db-tables` disabled on main container), the main API container will startup really fast, being only a matter of connecting to DB and Rabbit again.

Reply:
The drawback is that now studio api app will be less "self-contained". You'll need to know that before using it you need to run another up to set up the DB. 

Saying that, if the current approach caused us issues, then I'm with having this as an `initContainer`.

Reply:
My idea is to keep the index creation on by default!
https://github.com/livepeer/livepeer-com/blob/9ffcbf58e551cd0354a89bf54d78010a4a8b7d1c/packages/api/src/parse-cli.ts#L156-L161

So if you just run the API, it will create the indexes as usual. Only if you want to run the separate "create indexes" step that you would:
- disable `postgres-create-tables` on the main container
- define the `initContainer` (or whatever else on your stack) to create the indexes instead

Reply:
Ohh just another another idea too: In the future, maybe we can even make these not an `initContainer` but a step in the `livepeer-infra` deployment itself. So we run only once when deploying a new API version and all the API pods can start immediately without spending dozens of seconds creating indexes.